src/lib/command-helpers.mjs

```diff
@@ -22,6 +22,11 @@ const COMMON_QUERY_OPTIONS = {
     required: false,
     group: "API:",
   },
+  accountKey: {
+    type: "string",
+    description: "The account key to use when calling Fauna",
+    required: false,
+  },
```

Comment:
We called it out in the design phase as something that can be set by the user. I don't know the use case yet, but I know we have a UI page to allow users to create one of these keys so who knows how they might want to use those.

Reply:
We also said we want flag parity for everything. It's probably the "right" thing to do.

It would allow you to use the cli even if frontdoor login was failing but the rest of frontdoor was okay, right?